packages/@lwc/integration-types/package.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "@lwc/integration-types",
+    "private": true,
+    "version": "6.6.3",
+    "description": "Type validation for LWC packages",
+    "type": "module",
+    "scripts": {
+        "test": "tsc",
+        "playground": "rollup -c src/playground/rollup.config.js --watch"
+    },
+    "dependencies": {
+        "@lwc/rollup-plugin": "^6.6.3",
+        "lwc": "^6.6.3"
```

```suggestion
        "@lwc/rollup-plugin": "6.6.3",
        "lwc": "6.6.3"
```

We use exact versions for references between monorepo packages.